websites/B/Brave Search/presence.ts

```diff
@@ -1,6 +1,14 @@
 const presence = new Presence({
 		clientId: "1293341957141303307",
 	}),
+	strings = presence.getStrings({
+		search: "general.searchFor",
+		searchPrivate: "general.searchSomething",
+		home: "general.viewHome",
+		settings: "discord.settings",
+		help: "twitch.help",
```

Comment:
You cannot use strings from other presences anymore, either use other strings or make an own i18n file.